### ISSUE TITLE
max_execution_time is not available in MySQL 5.6

### ISSUE THREAD

**Issue by Daiyousei** [sixhop/AutoMySQLBackup#31]
When trying to backup a DB using the latest version (after commit f61b83fa219aab66a247f6202afa7bd017da2987), we get the following errors : 

```
ERROR 1193 (HY000) at line 1: Unknown system variable 'max_execution_time'
ERROR 1193 (HY000) at line 1: Unknown system variable 'max_execution_time'
```

max_execution_time should not be used on mysql 5.6. This variable had been introduced in MySQL 5.7.8

**Comment by AlejandroBOFH** [comment]
Also not available on MariaDB 10.3
According to https://mariadb.com/kb/en/library/system-variable-differences-between-mariadb-101-and-mysql-57/ it should be called max_statement_time

**Comment by inbaz** [comment]
I've added a statement to check only for mysql 5.7 and changed the variable to max_statement_time.
